features/networking/sctp.feature

```diff
@@ -0,0 +1,59 @@
+Feature: SCTP related scenarios
+
+  # @author weliang@redhat.com
+  # @case_id OCP-28757
+  @admin
+  @destructive
+  Scenario: Establish pod to pod SCTP connections		
+    Given I store the workers in the :workers clipboard
+    And the Internal IP of node "<%= cb.workers[1].name %>" is stored in the :worker1_ip clipboard
+    
+    Given I install load-sctp-module
+    # Installing sctp module will reboot all the nodes and it will more than 10 minutes
+    And I wait up to 800 seconds for the steps to pass:
+    """
+    Given 60 seconds have passed 
+    When I run the :get admin command with:
+      |resource | nodes |
```

Comment:
Missing space before ```resource```

Reply:
This line is deleted in the new version.